tests/zfs-tests/include/properties.shlib

```diff
@@ -14,7 +14,8 @@
 #
```

Comment:
For the Fixme on https://github.com/zfsonlinux/zfs/pull/9655/commits/b6a031903be732e54a223836b5b28b4f3ed834a3

```
 - adds zstd to compression test suite
 - adds zstd(-fast) to history_002_pos.ksh
 - adds random levels of zstd(fast) to history_002_pos.ksh

Signed-off-by: Allan Jude <allanjude@freebsd.org>
Signed-off-by: Sebastian Gottschall <s.gottschall@dd-wrt.com>
Signed-off-by: Kjeld Schouten-Lebbing <kjeld@schouten-lebbing.nl>
Signed-off-by: Michael Niewöhner <foss@mniewoehner.de>
```

"compression test suite"
- Might sound a bit vague, but if you actually know the testing related to compression (like I had to research for my PR), one would know what this refers too. Also, you could always point them to my refactor PR to point out the compression test suite.

Reply:
what's your suggestion? adding the PR#?

Reply:
No I wrote a new description, thats all... 
I just explained my new description ;)

Reply:
aah, yesss :)